src/components/questions_answers/QnA.jsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import Search from './QnA_Components/Search';
-// import exampleData from './exampleData';
+import QuestionList from './QnA_Components/QuestionList';
+import exampleData from './exampleData';
 
 export default function QnA() {
+  const questions = exampleData.results;
   return (
     <section>
       <h1>Questions &amp; Answers</h1>
       <Search />
-      <div>
-        <p>Q: Would it fit super skinny people?</p>
-        <p>A: Probably Not</p>
-      </div>
+      {questions.slice(0, 5).map((question) => (
+        <QuestionList question={question} />
```

Comment:
Add a `key` attribute whenever creating elements via iteration (console is showing warnings on load)